bitnami/redis/templates/redis-master-statefulset.yaml

```diff
@@ -1,4 +1,4 @@
-{{- if and .Values.cluster.enabled (not .Values.sentinel.enabled) }}
+{{- if (not .Values.sentinel.enabled) }}
```

Comment:
Shouldn't it be 

```
{{- if or (not .Values.cluster.enabled) (not .Values.sentinel.enabled) }}
```

Reply:
Applied discussed changes.

Now, when using the value combination of `cluster.enabled=false` and `sentinel.enabled=false` the chart deploys standalone redis and warns that said combination is not supported.

Reply:
sorry to reopen this discussion, but with this condition, `redis-master-statefulset.yaml` still not created when `cluster.enabled=false`. have you guys tested this before?